src/containers/Main.css

```diff
@@ -0,0 +1,5 @@
+.dark-mode {
+  background-color: #171c28;
+  color: white;
+  transition: "0.1s";
+}
```

Comment:
**Question**: Can't this fit into `index.css` or `app.css`?

Reply:
We can fit these into index and app CSS but let make sure we keep them as clean as possible and use the Main CSS as our css for all the Components.